docs/tequila-api.md

```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your o travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is you need to be registered and have your API key.
```

Comment:
`with the Tequila API is to be registered`

Reply:
Thanks, will change